outputs.tf

```diff
@@ -1,4 +1,47 @@
-output "random_pet_name" {
-  description = "The generated random pet name"
-  value       = random_pet.template.id
+output "teams" {
+  description = "Map of team names to their properties including ID and slug"
+  value = {
+    for name, team in github_team.default : name => {
```

Comment:
How is this different from `value = github_team.default`?

Reply:
My concern is that `value = github_team.default` would output ALL attributes of the resource, while the current format provides a cleaner output by explicitly selecting the most important (IMO) attributes. Does that make sense?